index.html

```diff
@@ -217,5 +217,53 @@ <h2 class="skill-name">Skills</h2>
 			</li>
 		</ul>
 	</section>
+	<section  class="contact-container" id="footer-container-id">
+		<div class="contact-form">
+			<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>		
+			<form action="https://formspree.io/f/xknagbpo" method="post">
+				<ul class="contact-elements">
+					<li class="text-input">
+						<input type="text" maxlength="30" placeholder="Full name" required>
+					</li>
+					<li>
+						<input type="email" placeholder="Email Address" required>
+					</li>
+					<li>
+						<textarea  type="text" maxlength="500" placeholder="Enter Text Here" cols="30" rows="10"></textarea>
```

Comment:
- [ ] Your form doesn't submit because you are missing the `name` attributes in all your input elements. I recommend that you fix this to allow the form submit successfully.

![image](https://user-images.githubusercontent.com/29071984/213035535-a6b91c62-65f7-43bc-bd7d-1332deeea3ea.png)